src/components/application_manager/test/commands/mobile/perform_audio_pass_thru_test.cc

```diff
@@ -96,6 +100,19 @@ class PerformAudioPassThruRequestTest
     return msg;
   }
 
+  MessageSharedPtr CreateHMIMessageSO(int32_t code, uint32_t cmd) {
+    MessageSharedPtr msg = CreateMessage(smart_objects::SmartType_Map);
+    (*msg)[am::strings::params][am::hmi_response::code] = code;
+    (*msg)[am::strings::msg_params][am::strings::cmd_id] = cmd;
+
+    return msg;
+  }
+
+  void CreateImageFile(std::string name) {
+    const std::string kFullFilePath = file_system::CurrentWorkingDirectory();
```

Comment:
@tkireva , `kFullFilePath` is not global. Please rename it to `full_file_path`
see [here](https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide)